### ISSUE TITLE
Create pop-up to select monthly report to create - CHW

### ISSUE THREAD

**Issue by msschroeder** [OpenSRP/opensrp-client-reporting#66]
- [ ] When you click "Start New Report", a pop up (figure 1 below) will appear allowing you to select any available report for creation. 
- [ ] If you have previously created/submitted that monthly report, it will no longer be listed here.
- [ ] Monthly report options are listed in the format YYYY-MM in reverse chronological order

![image5](https://user-images.githubusercontent.com/20777928/75348524-12bdaf00-58b4-11ea-9b28-0b466440e5f1.png)
Figure 1

- [ ] Once you select the month for the report, the report is generated and you are taken to the draft monthly report view. 

@manutarus @paulinembabu 
NOTE: If the user clears the cache, after making submissions and resync to the server, will I still be able to submit previous reports because I regenerated the daily reports? This should not be a problem.
NOTE: If the user clears the deletes the data, after making submissions and resync to the server, will I still be able to submit previous reports because I regenerated the daily reports? Yes - If they clear the data, there is a possibility to resubmit an report. What would be the technical implications to create a way to resync the reports back to the device?

**Comment by msschroeder** [comment]
@paulinembabu Can you and @manutarus address the questions in the notes at the bottom.

**Comment by msschroeder** [comment]
QA March 18:
lgtm - We can close the issue when the questions are addressed.

**Comment by paulinembabu** [comment]
@msschroeder the question here is unclear.

**Comment by msschroeder** [comment]
@paulinembabu  These questions: 

NOTE: If the user clears the cache, after making submissions and resync to the server, will I still be able to submit previous reports because I regenerated the daily reports? This should not be a problem. @paulinembabu Is this correct?

NOTE: If the user clears the deletes the data, after making submissions and resync to the server, will I still be able to submit previous reports because I regenerated the daily reports? Yes - If they clear the data, there is a possibility to resubmit an report. 

@paulinembabu What would be the technical implications to create a way to resync the reports back to the device?
